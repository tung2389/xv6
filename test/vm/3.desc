Test to write after unprotect