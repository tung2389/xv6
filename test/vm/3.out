XV6_TEST_OUTPUT Written to 0x1000
